pkg/client/retrieve.go

```diff
@@ -125,8 +131,9 @@ func UntarAll(reader io.Reader, destFile, prefix string) (filenames []string, re
 
 		for i := range b {
 			if b[i] != 0 {
-				returnErr = fmt.Errorf("non-zero data %v read after tar EOF", b[i])
-				return
+				//returnErr = fmt.Errorf("non-zero data %v read after tar EOF", b[i])
+				//return
+				fmt.Fprintf(os.Stderr, "non-zero data %v read after tar EOF", b[i])
```

Comment:
Need to revert this change. Originally I thought there was a tar difference on Windows so I wanted to see this data that was left around. In the end it was just an error on my part.